### ISSUE TITLE
using qlora: error:"CodeT5pEncoderDecoderModel does not support gradient checkpointing"

### ISSUE THREAD

**Issue by hot-zhy** [salesforce/CodeT5#169]
Hello! I use qlora to fine-tune codeT5p, but it errors: CodeT5pEncoderDecoderModel does not support gradient checkpoint.

Then I remove the parameter in function train:
![e0add5c3ba74a166c2957f6fd881c01](https://github.com/salesforce/CodeT5/assets/100272100/c9a3708c-b0d3-41e3-9e65-3415abfc0679)
but it doesn't work and it still errors:
![1dd6dc9ecebe31875c03b00bdc43067](https://github.com/salesforce/CodeT5/assets/100272100/2f839d29-165c-4abf-8931-804057a563f8)
What should i do? Can codeT5 use qlora to do SFT?
